scalafmt-cli/src/main/scala/org/scalafmt/cli/WriteMode.scala

```diff
@@ -8,5 +8,13 @@ package org.scalafmt.cli
   */
 sealed trait WriteMode
 
-case object Override extends WriteMode
-case object Stdout extends WriteMode
+object WriteMode {
```

Comment:
Instead of `null` by default is's possible to create

`case object Uninitialized extends WriteMode` here.

I believe that work with it will be more controllable and easier than with `null`

Reply:
sure. the value is actually never used and is replaced by the default `Override` in `CliOptions.auto`.

Reply:
Switched to `Option` and used `None` here.